src/lib.rs

```diff
@@ -685,4 +692,23 @@ impl Reflection {
         use rspirv::binary::Disassemble;
         self.0.disassemble()
     }
+
+    pub fn get_entry_points(&self) -> Result<Vec<EntryPoint>> {
+        let reflect = &self.0;
+        reflect
+            .entry_points
+            .iter()
+            .map(|entry_point| -> Result<EntryPoint> {
+                let execution_model = get_operand_at!(entry_point, Operand::ExecutionModel, 0)?;
+                let id = get_operand_at!(entry_point, Operand::IdRef, 1)?;
+                let name = get_ref_operand_at!(entry_point, Operand::LiteralString, 2)
+                    .map(Clone::clone)?;
```

Comment:
I think that's just `.cloned()`? But it should also be fine to return `EntryPoint<'_>` containing a borrowed `name: &'a str` with a lifetime on `&self`.

Reply:
Hah, thanks. DescriptorInfo is holding a string, so I followed the established way.